email_notification/sender/email_notification_context.py

```diff
@@ -88,11 +87,7 @@ def from_reservation(reservation: Reservation) -> "EmailNotificationContext":
         """Build context from reservation"""
         context = EmailNotificationContext()
 
-        if not reservation.reservee_type or reservation.reservee_type == CustomerTypeChoice.INDIVIDUAL.value:
-            context.reservee_name = f"{reservation.reservee_first_name} {reservation.reservee_last_name}"
-        else:
-            context.reservee_name = reservation.reservee_organisation_name
-
+        context.reservee_name = reservation.reservee_name
```

Comment:
This is used in the salutation of the email text ("Hi FirstName LastName") so defaulting to reservation.name would be undesirable.

Reply:
Changed these few rows back.